engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -337,26 +354,44 @@ public Vector3f centerPointForNormal(Vector3f normal) {
         return new Vector3f();
     }
 
+    /**
+     * @return the x coordinate of the min point
+     */
```

Comment:
No benefit in repeating all the minX, minZ in the javadoc.
AABB.minX() returns the min x coordinate obviously :)

same for all the following ones